runfinch/finch-daemon
# feat: Add support for socket Activation

Issue #, if available:

*Description of changes:*
This enables finch-daemon to be activated by connection to a configured socket also any request to the socket would wait for the daemon to be running preventing any race condition

*Testing done:*
1. Configure socket config file
2. stop finch-service
3. Send request to finch socket and confirm finch service is started.


- [ ] I've reviewed the guidance in CONTRIBUTING.md


#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
